tests/robot-tests/tests/general_public/permalink_prod_1.robot

```diff
@@ -43,10 +43,10 @@ Validate table
     user checks table cell contains    1    3    148,820
 
 Validate footnotes
-    user checks page contains element
-    ...    xpath://h3[text()="Footnotes"]/../ol/li[text()="State-funded secondary schools include city technology colleges and all secondary academies, including all-through academies and free schools."]
-    user checks page contains element
-    ...    xpath://h3[text()="Footnotes"]/../ol/li[text()="x - 1 or 2 pupils, or a percentage based on 1 or 2."]
+    user checks page contains element    //h3[@class="govuk-heading-m"]
```

Comment:
I think a better selector here would be `testid:footnotes`. The same change can be made in`permalink_prod_2` and `permalink_prod_3`.

While `//h3[@class="govuk-heading-m"]` it is unique on the page, there are other instances of tags with the class "govuk-heading-m", and if one of those turned into a h3 tag, then this test would always pass. 

Also, if additional class attribute were added to the h3 tag - i.e. `<h3 class="govuk-heading-m something-else">` - then this would also fail. If you want to selecting on a class, it's normally easier to use a CSS selector, like `css:h3.govuk-heading-m`. This is better because if another class was added, the CSS selector would still select work, while an xpath selector just compares the whole class string. 

You could do `//h3[contains(@class, 'govuk-heading-m')]`, but that has the issue that if someone adds a new tag to the service that looks like `<h3 class="govuk-heading-m-bold">` the selector would also find that element. So if you want to select on class, CSS is generally more robust.